mkdocs/structure/toc.py

```diff
@@ -2,15 +2,16 @@
 Deals with generating the per-page table of contents.
 
 For the sake of simplicity we use an existing markdown extension to generate
-an HTML table of contents, and then parse that into the underlying data.
+a JSON table of contents, and then store it as AnchorLinks to maintain
```

Comment:
That's not JSON. Its Python lists and dics, which (mostly) share there syntax with JSON. However, there are a few subtle differences. While those subtitles may not actually present themselves, it would be misleading to call it JSON.

Reply:
Whoops, somehow I got it into my head that `.toc_tokens` was being parsed from JSON somewhere, which obviously doesn't make any sense now that I think about it! Fixed.